clusters/clusters.json

```diff
@@ -0,0 +1,32 @@
+{
+    "shared": {
+        "time_limit": "00:30:00",
```

Comment:
AFAIK, this is different for every cluster. Maybe 1h is actually closer to a default?